packages/model-generator/API.md

```diff
@@ -196,7 +196,7 @@ export type ModelsGenerationParameters = {
 export type TypesGenerationParameters = {
     target: TypesTarget;
     multipleSwiftFiles?: boolean;
-};
+} & Pick<DocumentGenerationParameters, 'maxDepth' | 'typenameIntrospection'>;

```

Comment:
Wouldn't it be better to just declare two properties inline?
It takes additional jump for somebody reading this declaration to understand what `TypesGenerationParameters` is if we use `Pick`

Reply:
I did this to keep the types in sync, but I suppose the TypeScript build will already handle that with inline properties as well.